templates/files/partials/file_list.html

```diff
@@ -26,3 +26,21 @@ <h3 class="text-lg font-medium">{{ files_title|default:"FILES" }}</h3>
   {% endfor %}
 </ul>
 </div>
+
+<script>
```

Comment:
Suggestion for simpler ways to watch the `<ul>` for changes to its children elements welcome.